pkg/refactor/database/db_schema.sql

```diff
@@ -178,33 +178,41 @@ CREATE TABLE IF NOT EXISTS "staging_events" (
     "type" text NOT NULL,
     "cloudevent" text NOT NULL,
     "namespace_id" uuid NOT NULL,
+    "namespace" text,
     "namespace_name" text,
     "received_at" timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
     "created_at" timestamptz NOT NULL,
     "delayed_until" timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
     FOREIGN KEY ("namespace_id") REFERENCES "namespaces"("id") ON DELETE CASCADE ON UPDATE CASCADE,
+--    FOREIGN KEY ("namespace") REFERENCES "namespaces"("name") ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
We should discuss in our meeting if it's possible to remove the namespace_id entirely at this time.